### ISSUE TITLE
Have Verk start as a child the user's supervision tree. 

### ISSUE THREAD

**Issue by jeregrine** [edgurgel/verk#58]
```
16:46 <jeregrine> is there any reason why I *shouldn't* manually start an application outside of my mix.exs? Say I want to make sure Repo is started before an application?
16:48 <ericmj> jeregrine: it sounds weird to have a dependency depend on your app
16:48 <jeregrine> ericmj: yea it is. We're using the worker queue "verk" and the worker starts jobs which require the Repo before the repo has started
16:48 <jeregrine> its kind of a race condition
16:49 <jeregrine> so whenever we boot we get a couple jobs that fail
16:49 <edgurgel> yeah we would need a way to “wait” till the system is up
16:49 <edgurgel> cause Verk itself holds the queues,workers.
16:49 <ericmj> maybe the verk supervisor should be started by the parent app
16:49 <edgurgel> so Verk starts and it tries to run the jobs as fast as possible
16:50 <edgurgel> yeah it would probably be a good solution
```

**Comment by edgurgel** [comment]
I just released a new version `0.11`. I'll work on this soon. Technically it's just a change in the way the app starts and README.md instructions.

**Comment by jeregrine** [comment]
@edgurgel in our app we configure all of our verk queues to have 0 concurrency for tests.

**Comment by amiel** [comment]
I am experiencing the same issue. Thanks for the fix, I look forward to trying it out.

**Comment by edgurgel** [comment]
I just released `0.12.0` with this change. https://github.com/edgurgel/verk/releases/tag/v0.12.0
